packages/graph/hash_graph/lib/graph/src/store/postgres/query/statement/select.rs

```diff
@@ -651,12 +651,8 @@ mod tests {
             r#"
             SELECT *
             FROM "entities"
-            LEFT JOIN "entities" AS "entities_0_0_0"
-              ON "entities_0_0_0"."entity_uuid" = "entities"."left_entity_uuid"
-            LEFT JOIN "entities" AS "entities_0_0_1"
-              ON "entities_0_0_1"."entity_uuid" = "entities"."right_entity_uuid"
-            WHERE ("entities_0_0_0"."entity_uuid" = $1) AND ("entities_0_0_0"."owned_by_id" = $2)
-              AND ("entities_0_0_1"."entity_uuid" = $3) AND ("entities_0_0_1"."owned_by_id" = $4)
+            WHERE ("entities"."left_entity_uuid" = $1) AND ("entities"."left_owned_by_id" = $2)
+              AND ("entities"."right_entity_uuid" = $3) AND ("entities"."right_owned_by_id" = $4)
```

Comment:
This is a really common query as it identifies a link by it's left/right entity. The join statement is now much shorter.

Reply:
Yay! :sparkles: